bahmni-docker/docker-compose.yml

```diff
@@ -26,20 +26,26 @@ services:
       - '443:443'
     logging: *log-config
 
+  bahmni-config:
+    image: '${CONFIG_IMAGE_NAME:?}:${CONFIG_IMAGE_TAG:?}'
```

Comment:
no profile attribute, because this has to be started in all cases?

Reply:
Yes! Config will be shared by services from different profiles